src/platforms/common/data-management/event-grouping/fingerprint-rules.mdx

```diff
@@ -247,4 +275,14 @@ This fills in the value of a tag into the fingerprint, which can, for instance,
 message:"connection error*" -> connection-error, {{ tags.server_name }}
 ```
 
+### `{{ message }}`
+
+This fills in the message of the event (similar to `error.value` but for
+captured messages). Note that this can produce really bad groups when
+messages are frequently changing:
```

Comment:
Minor wording suggestion:
```suggestion
This fills in the message of the event (similar to `error.value` but for
captured messages). Note that this can produce groups with poor data quality if
messages are changing frequently:
```